BetsApi/Controllers/BetQuoteController.cs

```diff
@@ -0,0 +1,114 @@
+﻿using AutoMapper;
+using Domain.Dto.BetQuote;
+using Domain.Dto.Bets;
+using Domain.Dto.BetRequest;
+using Microsoft.AspNetCore.Mvc;
+using Services.Interfaces;
+
+namespace BetsApi.Controllers
+{
+    [ApiController]
+    [Route("/api/[controller]")]
+    public class BetQuoteController : Controller
+    {
+        private readonly IBetsService _betService;
+        private readonly IBetQuoteService _betQuoteService;
+        private readonly IMapper _mapper;
+
+        public BetQuoteController(IBetsService betService, IBetQuoteService betQuoteService, IMapper mapper)
+        {
+            _betService = betService;
+            _betQuoteService = betQuoteService;
+            _mapper = mapper;
+        }
+
+        [HttpPost("")]
+        public async Task<IActionResult> PlaceBetAsync(CreateBetRequest betRequest)
+        {
+            CreateBetsDto bet = betRequest.Bet;
+            CreateBetQuotesDto quote = betRequest.BetQuote;
+            BetsDto createdBet = await _betService.CreateAsync(bet);
+            Console.WriteLine("Pana aici e ok");
+            if (createdBet == null)
+                return BadRequest(ModelState);
+
+            BetQuoteDto createQuote = await _betQuoteService.CreateAsync(quote, createdBet.Id);
+            BetRequestDto betRequestDto = new BetRequestDto{bet = createdBet ,betQuote = createQuote};
+            return Ok(betRequestDto);
+        }
+
+        [HttpGet("{id}")]
+        public async Task<IActionResult> GetBetQuoteByIdAsync(Guid id)
```

Comment:
let's simplify the api and expose the bet with the quotes together. I think it would make more sense.